mubench.reviewsite/tests/StoreDetectorTest.php

```diff
@@ -21,6 +21,7 @@ function test_create_detector()
         "result": "failure",
         "runtime": 30.1,
         "number_of_findings": 0,
+        "-custom-stat-": "-stat-val-",
```

Comment:
Why do you need to touch this test? Storing detectors has nothing to do with storing run info, has it?